scripts/linux_installer.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import platform
+
+from installer import Installer
+
+class LinuxInstaller(Installer):
+    def __init__(self):
+        super().__init__()
+
+        self.THYRA_INSTALL_FOLDER_PATH = "/usr/local/bin"
+        self.THYRA_SERVER_FILENAME = "thyra-server"
+        self.THYRA_APP_FILENAME = "thyra-app"
+        self.MKCERT_FILENAME = "mkcert"
+
+        self.THYRA_SERVER_URL = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux_amd64"
+        self.THYRA_APP_URL = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_linux-amd64"
+        self.MKCERT_URL = "https://dl.filippo.io/mkcert/latest?for=linux/amd64"
+
+    def configureNetworkManager(self):
+        logging.info("Configuring NetworkManager...")
+        self.executeCommand("sudo cp /etc/NetworkManager/NetworkManager.conf /etc/NetworkManager/NetworkManager.conf.backup_thyra_install", True)
+        stdout, _stderr = self.executeCommand("grep '^dns=' /etc/NetworkManager/NetworkManager.conf", True, allow_failure=True)
```

Comment:
if you want, you can name this unsused variable `_`. but I understand if you want to be clear about what this variable is and that we deliberately don't want to use it. (this remark apply for some other cases in the code)

```suggestion
        stdout, _ = self.executeCommand("grep '^dns=' /etc/NetworkManager/NetworkManager.conf", True, allow_failure=True)
```